client.cpp

```diff
@@ -1593,5 +1593,6 @@ void run_stats::print(FILE *out, bool histogram, const char * header/*=NULL*/,
         }
         if (jsonhandler != NULL){ jsonhandler->close_nesting();}
     }
+    if (jsonhandler != NULL){ jsonhandler->close_nesting();}
```

Comment:
Add comment:
"This close_nesting closes either:
jsonhandler->open_nesting(header); or
jsonhandler->open_nesting("UNKNOWN STATS");
From the top (beginning of function).